### PR TITLE
Added Python Resources to the repository

### PR DESCRIPTION
**Pull request by aaqaishtyaq**
Kindly review the resources and modify if necessary.

### CODE REVIEW

File: Python.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  - [Python Framework Comparison: Django vs. Pyramid](https://www.codementor.io/python/tutorial/django-vs-pyramid-python-framework-comparison)
+  - [How to Set up NumPy on a 64 bit Windows OS](https://www.codementor.io/numpy/tutorial/installing-numpy-64-bit-windows)
+  - [Python Q&A with #1 Stack Overflow Python Expert](https://www.codementor.io/python/tutorial/stack-overflow-python-expert-answers-questions)
+  - [Stack Overflow Legend Martijn Pieters: Python Optimization and How it Can Affect Your Code](https://www.codementor.io/python/tutorial/stack-overflow-martijn-pieters-python-optimization)
+  - [6 Useful Python Libraries Recommended by #1 Stack Overflow Answerer](https://www.codementor.io/python/tutorial/6-useful-python-libraries)
+  - [Martijn Pieters on the Future of Django](https://www.codementor.io/python/tutorial/martijn-pieters-future-django)
+  - [Python Internals: Codementor Office Hours with Martijn Pieters](https://www.codementor.io/python/tutorial/python-internals-codementor-office-hours-martijn-pieters)
+  - [Building a Simple Snake Game with Python](https://www.codementor.io/python/tutorial/build-snake-game-using-curses)
+
+
```
Comment (juggernaut451) [inline Python.md]: remove extra line

File: Python.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  - [Fetching an RSS feed and converting it into a json response](https://www.codementor.io/tips/3493772831/fetching-an-rss-feed-and-converting-it-into-a-json-response)
+  - [Opencv: display contours](https://www.codementor.io/tips/4930721378/opencv-display-contours)
+  - [Django form choices loaded from database are not updatedCan't figure out what's wrong with my method](https://www.codementor.io/tips/4933721348/can-t-figure-out-what-s-wrong-with-my-method)
+  - [Different initial values in a Django formset](https://www.codementor.io/tips/0731824832/different-initial-values-in-a-django-formset)
+  - [Why does python's struct think little-endian and big-endian imply different lengths?](https://www.codementor.io/tips/0818274433/why-does-python-s-struct-think-little-endian-and-big-endian-imply-different-lengths)
+  - [Comparing two class types in python](https://www.codementor.io/tips/6130438872/comparing-two-class-types-in-python)
+  - [How can I get Travis-CI to recognize and open external files during testing](https://www.codementor.io/tips/8102473038/how-can-i-get-travis-ci-to-recognize-and-open-external-files-during-testing)
+
```
Comment (juggernaut451) [inline Python.md]: remove extra line

File: Python.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  - [Comparing two class types in python](https://www.codementor.io/tips/6130438872/comparing-two-class-types-in-python)
+  - [How can I get Travis-CI to recognize and open external files during testing](https://www.codementor.io/tips/8102473038/how-can-i-get-travis-ci-to-recognize-and-open-external-files-during-testing)
+  
+  
+## Videos
+  - [Carte-Blanche, a Django Permission Framework for Rapid Prototyping, with its creator, Eric Neuman ](https://www.codementor.io/officehours/6962831504/carte-blanche-a-django-permission-framework-for-rapid-prototyping-with-its-creator-eric-neuman)
+  - [Introduction to Machine Learning & NLP: Building a Spam Classifier](https://www.codementor.io/officehours/1385095426/building-a-spam-classifier)
+  - [Stack Overflow Legend Martijn Pieters: Python Optimization and How it Can Affect Your Code](https://www.codementor.io/officehours/9015644327/stack-overflow-legend-martijn-pieters-python-optimization-and-how-it-can-affect-your-code)
+
+
```
Comment (juggernaut451) [inline Python.md]: remove extra line

File: Python.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+## Courses
+  - [Live Python Class](https://www.codementor.io/classes/learn-python-live)
+  - [MIT Introduction to Computer Science and Programming](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-00sc-introduction-to-computer-science-and-programming-spring-2011/)
+  - [Programming for Everybody (Python)](https://www.coursera.org/course/pythonlearn)
+  - [Google's Python Class](https://developers.google.com/edu/python/?hl=de-DE&csw=1)
+  - [Intro to Computer Science](https://www.udacity.com/course/ud036)
+  - [Code School's Try Python](https://www.codeschool.com/courses/try-python)
+
+
```
Comment (juggernaut451) [inline Python.md]: remove extra line

**Review by juggernaut451 — CHANGES_REQUESTED** [review]

**Comment by aaqaishtyaq** [comment]
Removed extra lines

File: Python.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
+# Python 
+Python is an easy to learn language many beginners to coding choose as their first programming language and it is not limited to web development, as you can build games and applications for academical research with it. If you are new to programming or simply interested in learning Python, here are some resources you can use.
+## Platform for Python Tutorials
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  - [Bucky's Room](https://buckysroom.org/videos.php?cat=98)
+  - [Think Python](http://www.greenteapress.com/thinkpython/)
+  - [Learn Python](http://www.learnpython.org/)
+  - [Python for You and Me](http://pymbook.readthedocs.org/en/py3/)
+  - [Learn Python the Hard Way](http://learnpythonthehardway.org/book/)
+## General Topics for Beginners
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  - [Python Framework Comparison: Django vs. Pyramid](https://www.codementor.io/python/tutorial/django-vs-pyramid-python-framework-comparison)
+  - [How to Set up NumPy on a 64 bit Windows OS](https://www.codementor.io/numpy/tutorial/installing-numpy-64-bit-windows)
+  - [Python Q&A with #1 Stack Overflow Python Expert](https://www.codementor.io/python/tutorial/stack-overflow-python-expert-answers-questions)
+  - [Stack Overflow Legend Martijn Pieters: Python Optimization and How it Can Affect Your Code](https://www.codementor.io/python/tutorial/stack-overflow-martijn-pieters-python-optimization)
+  - [6 Useful Python Libraries Recommended by #1 Stack Overflow Answerer](https://www.codementor.io/python/tutorial/6-useful-python-libraries)
+  - [Martijn Pieters on the Future of Django](https://www.codementor.io/python/tutorial/martijn-pieters-future-django)
+  - [Python Internals: Codementor Office Hours with Martijn Pieters](https://www.codementor.io/python/tutorial/python-internals-codementor-office-hours-martijn-pieters)
+  - [Building a Simple Snake Game with Python](https://www.codementor.io/python/tutorial/build-snake-game-using-curses)
+## Data Science with Python
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+  - [Data Frames with Python & R](https://www.codementor.io/python/tutorial/python-vs-r-for-data-science-data-frames-i), [Part 2](https://www.codementor.io/python/tutorial/python-vs-r-data-science-data-frames-ii)
+  - [Exploratory Data Analysis](https://www.codementor.io/python/tutorial/data-science-python-r-exploratory-data-analysis-visualization)
+  - [Dimension Reduction & Clustering](https://www.codementor.io/python/tutorial/data-science-python-pandas-r-dimensionality-reduction)
+  - [Sentiment Classification Using Linear Methods](https://www.codementor.io/python/tutorial/data-science-python-r-sentiment-classification-machine-learning)
+  - [Building a Wine Review & Recommendation Site with Django](https://www.codementor.io/python/tutorial/get-started-with-django-building-recommendation-review-app), [Part 2](https://www.codementor.io/python/tutorial/build-data-product-django-user-management), [Part 3](https://www.codementor.io/python/tutorial/build-data-products-django-machine-learning-clustering-user-preferences)
+## Python & Apache Spark
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+  - [Getting Data into the basic Spark data structure: Resilient Distributed Datasets](https://www.codementor.io/spark/tutorial/spark-python-rdd-basics), [Part 2](https://www.codementor.io/spark/tutorial/spark-python-data-aggregations)
+  - [Machine Learning Library (MLlib) Basic Statistics & Exploratory Data Analysis](https://www.codementor.io/spark/tutorial/mllib-basic-statistics-exploratory-data-analysis)
+  - [MLlib Logistic Regression](https://www.codementor.io/spark/tutorial/spark-mllib-logistic-regression)
+  - [MLlib Decision Trees](https://www.codementor.io/spark/tutorial/spark-python-mllib-decision-trees)
+  - [SQL & DataFrames](https://www.codementor.io/spark/tutorial/python-spark-sql-dataframes)
+  - [Building a Movie Recommendation Service with Apache Spark & Flask](https://www.codementor.io/spark/tutorial/building-a-recommender-with-apache-spark-python-example-app-part1), [Part 2](https://www.codementor.io/spark/tutorial/building-a-web-service-with-apache-spark-flask-example-app-part2)
+## Tips
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+  - [Fetching an RSS feed and converting it into a json response](https://www.codementor.io/tips/3493772831/fetching-an-rss-feed-and-converting-it-into-a-json-response)
+  - [Opencv: display contours](https://www.codementor.io/tips/4930721378/opencv-display-contours)
+  - [Django form choices loaded from database are not updatedCan't figure out what's wrong with my method](https://www.codementor.io/tips/4933721348/can-t-figure-out-what-s-wrong-with-my-method)
+  - [Different initial values in a Django formset](https://www.codementor.io/tips/0731824832/different-initial-values-in-a-django-formset)
+  - [Why does python's struct think little-endian and big-endian imply different lengths?](https://www.codementor.io/tips/0818274433/why-does-python-s-struct-think-little-endian-and-big-endian-imply-different-lengths)
+  - [Comparing two class types in python](https://www.codementor.io/tips/6130438872/comparing-two-class-types-in-python)
+  - [How can I get Travis-CI to recognize and open external files during testing](https://www.codementor.io/tips/8102473038/how-can-i-get-travis-ci-to-recognize-and-open-external-files-during-testing)
+## Videos
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+  - [Carte-Blanche, a Django Permission Framework for Rapid Prototyping, with its creator, Eric Neuman ](https://www.codementor.io/officehours/6962831504/carte-blanche-a-django-permission-framework-for-rapid-prototyping-with-its-creator-eric-neuman)
+  - [Introduction to Machine Learning & NLP: Building a Spam Classifier](https://www.codementor.io/officehours/1385095426/building-a-spam-classifier)
+  - [Stack Overflow Legend Martijn Pieters: Python Optimization and How it Can Affect Your Code](https://www.codementor.io/officehours/9015644327/stack-overflow-legend-martijn-pieters-python-optimization-and-how-it-can-affect-your-code)
+## Documentations
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+  - [Python Documentations](https://www.python.org/doc/)
+## Courses
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

File: Python.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+  - [Live Python Class](https://www.codementor.io/classes/learn-python-live)
+  - [MIT Introduction to Computer Science and Programming](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-00sc-introduction-to-computer-science-and-programming-spring-2011/)
+  - [Programming for Everybody (Python)](https://www.coursera.org/course/pythonlearn)
+  - [Google's Python Class](https://developers.google.com/edu/python/?hl=de-DE&csw=1)
+  - [Intro to Computer Science](https://www.udacity.com/course/ud036)
+  - [Code School's Try Python](https://www.codeschool.com/courses/try-python)
+## Developer Tools
```
Comment (juggernaut451) [inline Python.md]: Add a single line break before topic

**Review by juggernaut451 — CHANGES_REQUESTED** [review]

**Comment by aaqaishtyaq** [comment]
Added one line break after each topic

**Comment by juggernaut451** [comment]
LGTM. Merging it

**Review by juggernaut451 — APPROVED** [review]

**Pull request merged**